### PR TITLE
Support 5G on the Helium network

### PR DESCRIPTION
**Pull request by zer0tweets**
Rendered view: https://github.com/helium/HIP/blob/4455cbfb575eefcf4ece4ad1a91c6b0a14d09f90/0000-template.md

### CODE REVIEW

File: 0000-template.md [hunk 1]
```diff
@@ -1,97 +1,126 @@
...
-- It should be reasonably clear how the proposal would be implemented.
+- Types of wireless protocol i.e. LTE, 5G, Wi-Fi etc. 
+- Specific location of the access point transmitting the packets, since data prices vary per geography
 
-- Provide representative examples that show how this proposal would be commonly
-  used.
+In the future, as we grow and expand the network, we may want to have different conversion ratios, but for the purposes of the initial contemplated pilot, we’d like to focus on introducing just a conversion ratio that would denominate economics of a CBRS network. To figure out the initial cost of CBRS Data Credit we looked at the following guiding factors: 
 
-- Corner cases should be dissected by example.
+- Cheap prepaid LTE data plans in US sell for $1.50/Gb (Mint Mobile 15Gb bundle)
```
Comment (mfalkvidd) [inline 0000-template.md:37]: Maybe add a reference to this price point? I am unable to find it. The 15GB/month plan costs $45 ($3/GB) when I check https://www.mintmobile.com/plans/

File: 0000-template.md [hunk 2]
```diff
@@ -1,97 +1,126 @@
...
+- There are 2,000 CBRS nodes on the network.  
+- 50 HNT would be split proportionally to the CBRS and LoRaWan Hotspots who did the work of passing data and the remaining 1,624,950 HNT from the DC allocation would be distributed wit
+  - 1,624,950 - (1,624,950*30%) = 487,470 will be distributed ratably among the operators the CBRS LTE network 
+  - 1,624,950 - 487,470 = 1,137,480 will be distributed ratably among the Challengers, Witnesses and Challengees of the LoRaWan network 
+
+[This model](https://docs.google.com/spreadsheets/d/1-ARuH5-JQveDk_yd0Dn3JrTTrD6f6xUSD7MkhZJkHC0/edit?usp=sharing) enables us to play with various scenarios for above: 
+
+##Outcome of new scheme
+
+- Introduces general framework and system variables (new DC vars and new economic model) for expanding Helium network beyond LoRoWan use case
```
Comment (mfalkvidd) [inline 0000-template.md:117]: Typo: LoRoWAN

File: 0000-template.md [hunk 3]
```diff
@@ -1,97 +1,126 @@
...
+- Types of wireless protocol i.e. LTE, 5G, Wi-Fi etc. 
+- Specific location of the access point transmitting the packets, since data prices vary per geography
 
-- Provide representative examples that show how this proposal would be commonly
-  used.
+In the future, as we grow and expand the network, we may want to have different conversion ratios, but for the purposes of the initial contemplated pilot, we’d like to focus on introducing just a conversion ratio that would denominate economics of a CBRS network. To figure out the initial cost of CBRS Data Credit we looked at the following guiding factors: 
 
-- Corner cases should be dissected by example.
+- Cheap prepaid LTE data plans in US sell for $1.50/Gb (Mint Mobile 15Gb bundle) 
+- Comcast charges $10 for each $50Gb
```
Comment (mfalkvidd) [inline 0000-template.md:38]: Typo: this text uses **Gb** (=gigabit) throughout the document. Most data plans are marketed in **GB** (=gigabytes).

File: 0000-template.md [hunk 4]
```diff
@@ -1,97 +1,126 @@
...
-# Deployment Impact
-[deployment-impact]: #deployment-impact
+Unlike LoRaWan access points, cellular radios do not have the ability to challenge and witness each other in the same way. Cellular networks are architected to facilitate communication exclusively between a base station and a UE (user equipment) with a sim card in it. Base stations themselves do not have a sim card and cannot emulate a UE, which makes it impossible for one base station to “witness” the RF “challenge” issued by another base station.  
 
-Describe how this design will be deployed and any potential imapact it may have on
-current users of this project.
+The good news, however, is that unlike in LoRaWan networks, cellular networks already have a large number of devices that are continuously seeking for a cell tower to attach to. To validate that the CBRS cell is on and, at the same time, to estimate the usefulness of a particular CBRS cell on the network, Helium miner attached to the cell will need to perform the following tasks: 
 
-- How will current users be impacted?
+- Continuously collect data on the unique number of IMSIs attempting to attach to the PLMN(s) broadcasted by the cell.
```
Comment (mfalkvidd) [inline 0000-template.md:68]: Might need to be clear on how to protect privacy. Depending on juridistiction, IMSIs will be considered personal identifiable information (PII).

**Review by mfalkvidd — COMMENTED** [review]
Thanks for introducing this. I really like the concept. Some minor comments to improve the text.

File: 0000-template.md (hunk 2 above)
Comment (jamiew) [reply 0000-template.md:117]: Fixed this one

File: 0000-template.md (hunk 3 above)
Comment (jamiew) [reply 0000-template.md:38]: Also fixed

**Review by jamiew — APPROVED** [review]
LGTM, merging for discussion! Welcome HIP27

**Pull request merged**